src/layouts/Home.tsx

```
@@ -78,18 +82,18 @@ export function Home({ children, type, st, isAuth }: HomeProps) {
+                      <UserStatsBanner />
+                    ) : (
+                      <BannerCarousel />
+                    )}
                   </>
                 )}
                 {type === 'listing' && (
                   <>
-                    <NavTabs />
-                    {!authenticated ? <BannerCarousel /> : <UserStatsBanner />}
+                    {authenticated ? <UserStatsBanner /> : <BannerCarousel />}
                   </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)